browser-sign-in/app/src/main/java/com/okta/android/samples/browser_sign_in/ServiceLocator.java

```diff
@@ -102,29 +97,4 @@ public static SmartLockHelper provideSmartLockHelper() {
     public static OktaProgressDialog provideOktaProgressDialog(Context context) {
         return new OktaProgressDialog(context);
     }
-
-    public static RxWebAuthClient provideRxWebAuthClient(Context context) {
```

Comment:
Updating to the latest okta SDK made it such that implementing this was difficult due to lacking constructors.

This stuff wasn't actually used in the sample, so it was easiest just to remove it.

If you have another route you'd like me to take, just let me know.